File: code_generator.py
import random

# Word banks for code combinations
first_word_bank = [
    'PEANUT',
    'EXIT',
    'TUNE',
    'BONFIRE',
    'JAW',
    'MOTHER',
]

second_word_bank = [
    'ASTRONAUT',
    'MONSTERS',
    'SOUR',
    'BLEND',
    'PLANE',
]

third_word_bank = [
    'COCONUT',
    'SUIT',
    'BLUE',
    'ROCKY',
    'MOON',
    'TREETOP',
]

fourth_word_bank = [
    'FIELD',
    'PLANE',
    'CLOUD',
    'NUKE',
    'SHOWER',
    'BUGS',
]

# Function to permeate every possible combination
def get_code_options():
    options = []

    for first_word in first_word_bank:
        for second_word in second_word_bank:
            for third_word in third_word_bank:
                for fourth_word in fourth_word_bank:
                    options.append(f'{first_word} {second_word} {third_word} {fourth_word}')

    return options

# Get options and shuffle the list
code_options = get_code_options()
random.shuffle(code_options)

# Print each code option line-by-line
for code in code_options:
    print(code)

File: code_generator.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

// Word banks for code combinations
var firstWordBank = []string{
	"PEANUT",
	"EXIT",
	"TUNE",
	"BONFIRE",
	"JAW",
	"MOTHER",
}

var secondWordBank = []string{
	"ASTRONAUT",
	"MONSTERS",
	"SOUR",
	"BLEND",
	"PLANE",
}

var thirdWordBank = []string{
	"COCONUT",
	"SUIT",
	"BLUE",
	"ROCKY",
	"MOON",
	"TREETOP",
}

var fourthWordBank = []string{
	"FIELD",
	"PLANE",
	"CLOUD",
	"NUKE",
	"SHOWER",
	"BUGS",
}

// Function to permeate every possible combination
func getCodeOptions() []string {
	var options []string

	for _, firstWord := range firstWordBank {
		for _, secondWord := range secondWordBank {
			for _, thirdWord := range thirdWordBank {
				for _, fourthWord := range fourthWordBank {
					options = append(options, fmt.Sprintf("%s %s %s %s", firstWord, secondWord, thirdWord, fourthWord))
				}
			}
		}
	}

	return options
}

func main() {
	// Get options and shuffle list
	codeOptions := getCodeOptions()

	rand.Seed(
		time.Now().UnixNano(),
	)

	rand.Shuffle(
		len(codeOptions),
		func(i, j int) { codeOptions[i], codeOptions[j] = codeOptions[j], codeOptions[i] },
	)

	// Log each code option line-by-line
	for _, code := range codeOptions {
		fmt.Println(code)
	}
}
